pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1490,6 +1490,11 @@ self: super: {
   gi-gtk-hs = assert super.gi-gtk-hs.version == "0.3.8.1"; self.gi-gtk-hs_0_3_9;
   gi-xlib = assert super.gi-xlib.version == "2.0.8"; self.gi-xlib_2_0_9;
 
+  # Readline uses Distribution.Simple from Cabal 2, in a way that is not
+  # compatible with Cabal 3
+  readline = unmarkBroken (
```

Comment:
If you're removed `broken = true` from `hackage-packages.nix`, you shouldn't need this `unmarkBroken` here.

Although you may want to remove `readline` from the list of broken packages in `configuration-hackage2nix.yaml`.

Reply:
Ah true, I will update this